services/core/measurements/infra/store_psql.go

```diff
@@ -309,3 +315,21 @@ func (s *MeasurementStorePSQL) ListDatastreams(filter measurements.DatastreamFil
 	page := pagination.CreatePageT(ds, cursor)
 	return &page, nil
 }
+
+func (s *MeasurementStorePSQL) GetDatastream(id uuid.UUID) (*measurements.Datastream, error) {
+	var ds measurements.Datastream
+	idB, _ := id.MarshalBinary()
+	err := s.db.Get(&ds, `
+		SELECT 
```

Comment:
Yep. for simple queries I tend to directly address the DB instead of using the builder. I only use the builder if the query is "Dynamic" i.e. if a where clause only appears sometimes.